packages/http/src/__tests__/client.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { createClientFromOperations } from '../client';
+import { mocker } from '../mocker';
+import { IHttpConfig } from '../types';
+
+describe('User Http Client', () => {
+  describe('with mocking set to true', () => {
+    describe('get a resource', () => {
+      let client: ReturnType<typeof createClientFromOperations>;
+
+      const config: IHttpConfig = {
+        mock: { dynamic: false },
+        validateRequest: true,
+        validateResponse: true,
+        securityChecks: true,
+      };
+
+      beforeAll(async () => {
+        client = await createClientFromOperations(
```

Comment:
still seems like the coverage might be low? 
`export { createClientFromResource, createClientFromString, createClientFromOperations };` vs only `createClientFromOperations` tested

Reply:
Clearly we're not there yet. I am trying to find a decent coverage to test this stuff.